DetectorDescription/Detectors/src/SimpleCylinder_geo.cxx

```diff
@@ -0,0 +1,53 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple cylinder using Tube to be used to define cylinder composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
```

Comment:
Same as above: Please remove the `using namespace` statements and replace with more specific statements.